@navikt/ds-css/accordion-menu/index.css

```diff
@@ -0,0 +1,86 @@
+html {
+  scroll-behavior: smooth;
+}
+
+.navds-accordion-menu__container {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.navds-accordion-menu__item {
+  position: relative;
+  padding: 1px 0;
+  margin: 0;
+}
+
+.navds-accordion-menu__title {
+  margin-bottom: 0.75rem;
+}
+
+.navds-accordion-menu__item--active::after {
```

Comment:
Jeg lurer på om det hadde vært enklere om `nav`-elementet ikke hadde noen padding, så elementene er fullbredde, og at man da kan løse aktiv-stil med er border-left. Da blir også :active og :focus markering fullbredde, som gir mer mening for meg. Jeg ser at i arbeid sin meny er elementene fullbredde.

Reply:
Fiksa :)